src/main/java/mekanism/common/integration/energy/fluxnetworks/FNIntegration.java

```diff
@@ -125,14 +125,15 @@ public boolean canExtract() {
     @Override
     public boolean canReceive() {
         //Mark that we can receive energy if we can insert energy
-        if (handler.insertEnergy(FloatingLong.ONE, Action.SIMULATE).smallerThan(FloatingLong.ONE)) {
+        //todo: is this right? shouldn't it be only if 1 FE works, this is in joules
+        if (handler.insertEnergy(1, Action.SIMULATE) == 1) {
```

Comment:
This should be `== 0`. And to answer the todo (other than the bug I mention at the start of this comment), this is meant to be in joules, because *technically* the `canReceive` and `canExtract` methods are if we support doing that ever. So this is our attempt at providing an accurate result to those methods.